osd_cluster_create/byo-vpc.adoc

```diff
@@ -0,0 +1,42 @@
+[id="byo-vpc"]
+= Managing a Bring-Your-Own Virtual Private Cloud (VPC)
+:context: byo-vpc
+include::modules/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+Configuring a cluster to include your own pre-configured Virtual Private Cloud (VPC) allows `*[WHAT]*`
+
+.Prerequisites
+* You have a configured VPC
+* You have a configured AWS account in the `osd-staging-2` organization
```

Comment:
what does `osd-staging-2` mean? May I know why user need the organization?

Reply:
This is specific to internal teams. I'm not sure why its showing here

Reply:
@dofinn Is there another way to word that they need to attach the AWS account to an organization? What should that organization have so that I can explain it rather than just the name?

Reply:
`osd-staging-2` is SREPs AWS Organization. This can then read as 
`You have a configured AWS account in your AWS organization` The name of this will be up to the CU